test/zdtm/static/cgroup04.checkskip

```diff
@@ -1,3 +1,10 @@
 #!/bin/bash
 
-! test -f /sys/fs/cgroup/cgroup.controllers
+test -f /sys/fs/cgroup/cgroup.controllers && exit 1
+
+grep -q 'cgroup rw,' /proc/mounts && exit 1
```

Comment:
> cgroup04 test needs full control over mem and devices cgroup hierarchies.

In our CI, all cgroups are mounted and they are controlled by systemd, but the test is running without any issues.